source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -176,8 +176,9 @@ bool ShaderRenderTester::validate(const mx::FilePath optionsFilePath)
     _shaderGenerator->getUnitSystem()->setUnitConverterRegistry(registry);
 
     mx::GenContext context(_shaderGenerator);
-    context.registerSourceCodeSearchPath(currentPath);
+    context.registerSourceCodeSearchPath(searchPath);
```

Comment:
What is the correct logic here? current path or search path.
Should current path for osl also be updated ?

Reply:
Since this is for testing, currentPath is good.